test/JDBC/parallel_query_jdbc_schedule

```diff
@@ -55,6 +55,7 @@ ignore#!#babel_collection
 ignore#!#binary-index-vu-verify
 ignore#!#pgr_select_into
 ignore#!#cast_eliminate-vu-verify
+ignore#!#BABEL-4312
```

Comment:
What's the error for parallel schedule ?

Reply:
Plan is the same but it’s printing an extra line of the gather node.

Reply:
In that case, we can either
1. create a separate test file for parallel schedule, or
2. create a test-only function that looks up given string from explain output (it requires some effort, but will be useful)

Reply:
Currently parallel schedule depends on jdbc_schedule, we can only exclude extra tests in parallel schedule, but cannot run one in normal jdbc and another in parallel...
We have a group of those tests with different query plan under parallel mode, all of them need further validation and improvement in our parallel test framework. So I think this can be tackled as a whole in future work, will file a Jira for this.